<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel_main</class>
 <widget class="QWidget" name="panel_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>507</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>507</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>507</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reloj</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #303030; 
    color: #e0e0e0; 
    font-size: 14px;
}

QPushButton {
    background-color: #2196f3;
    color: white;
    border: none;
    padding: 10px 20px;
    border-radius: 4px;
    font-size: 16px;
}

QPushButton:hover {
    background-color: #1976d2;
}

QPushButton:pressed {
    background-color: #1565c0;
}

QCheckBox {
    color: #e0e0e0;
}

QLabel {
    color: #e0e0e0;
    font-size: 24px;
    font-weight: bold;
}


#relojLbl {
    font-size: 48px;
    font-weight: bold;
    color: #bb86fc; 
}


#IblMensaje_2 {
    font-size: 16px;
    color: #cf6679; 
}


#panel_main {
    background-color: #1e1e1e; 
    border-radius: 8px;
    padding: 20px;
}

#panel_central {
    background-color: #272727; 
    border-radius: 4px;
    padding: 10px;
}</string>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>511</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="chck12H">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>144</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Formato 12h</string>
    </property>
   </widget>
   <widget class="QLabel" name="relojLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>333</width>
      <height>82</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>00:00:00 AM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnActivar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>144</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Editar Alarma</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>481</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="lblAlarma">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Alarmas:</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>441</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblMensaje">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #e0e0e0;
    font-size: 14px;
    font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
